lib/livebook/config.ex

```diff
@@ -35,6 +35,11 @@ defmodule Livebook.Config do
       value: "Audience (aud)",
       module: Livebook.ZTA.GoogleIAP
     },
+    %{
+      type: :livebook_teams,
+      name: "Livebook Teams",
+      module: Livebook.ZTA.LivebookTeams
```

Comment:
@josevalim any objection to renaming ZTA to Identity everywhere? I can do it separately.

Reply:
Let's worry about it later as there is still a lot in flux.